contracts/interchain-token-service/src/storage_types.rs

```diff
@@ -11,3 +13,10 @@ pub enum DataKey {
     InterchainTokenWasmHash,
     TokenId(BytesN<32>),
 }
+
+#[contracttype]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct TokenIdConfig {
```

Comment:
```suggestion
    TokenIdConfigKey(BytesN<32>),
}

#[contracttype]
#[derive(Clone, Debug, PartialEq, Eq)]
pub struct TokenIdConfigValue {
```
soroban key/value naming convention from what i've seen

Reply:
updated using config key/value